.github/workflows/ci.yml

```diff
@@ -0,0 +1,247 @@
+            echo "Downloaded server bundle for release-7.8 from releases.mattermost.com"
+        fi
+        tar xf server.tar.gz
+    - name: ci/Replace-web-app
+      run: |
+        mv mattermost/client client-original
+        tar xf mattermost-webapp.tar.gz
+        mv client mattermost/client
+    - name: ci/Re-add-products
+      run: |
+        mv client-original/products mattermost/client/products
+    - name: ci/Repackage server bundle
+      run: |
+        tar -zcf mattermost-enterprise-linux-amd64.tar.gz mattermost/
+    - name: ci/persist-artifact
+      uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+      with:
+        name: mattermost-enterprise-linux-artifact
+        path: mattermost-enterprise-linux-amd64.tar.gz
+        retention-days: 14
+    - name: ci/aws-configure
+      uses: aws-actions/configure-aws-credentials@07c2f971bac433df982ccc261983ae443861db49 # v1-node16
+      with:
+        aws-region: ${{ secrets.WEBAPP_AWS_REGION }}
+        aws-access-key-id: ${{ secrets.WEBAPP_AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.WEBAPP_AWS_SECRET_ACCESS_KEY }}
+    - name: ci/artifact-upload
+      run: |
+        aws s3 cp mattermost-enterprise-linux-amd64.tar.gz s3://${{secrets.AWS_S3_BUCKET_PRS}}/$REPO_NAME/commit/${{ github.event.pull_request.head.sha || github.sha }}/ --cache-control no-cache
+  build-docker:
+    runs-on: ubuntu-22.04
+    needs: prepare-docker-build
+    steps:
+    - name: cd/configure-env-variables
+      id: vars
+      # Extract short-sha as TAG, and repository name as REPO_NAME
+      run: |
+        echo "TAG=`echo ${{ github.event.pull_request.head.sha || github.sha }} | cut -c1-7`" >> $GITHUB_ENV
+        echo "REPO_NAME=${{github.event.repository.name}}" >> $GITHUB_ENV
+    - name: cd/Login to Docker Hub
+      uses: docker/login-action@3da7dc6e2b31f99ef2cb9fb4c50fb0971e0d0139 # v2.1.0
+      with:
+        username: ${{ secrets.DOCKERHUB_DEV_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_DEV_TOKEN }}
+    - name: cd/Setup Docker Buildx
+      uses: docker/setup-buildx-action@11e8a2e2910826a92412015c515187a2d6750279 # v2.4
+    - name: cd/docker-build-push
+      env:
+        DOCKER_CLI_EXPERIMENTAL: enabled
+      run: |
+        wget https://raw.githubusercontent.com/mattermost/mattermost-server/release-7.8/build/Dockerfile
+        wget https://raw.githubusercontent.com/mattermost/mattermost-server/release-7.8/build/entrypoint.sh
+        chmod +x entrypoint.sh
+        export MM_PACKAGE=https://pr-builds.mattermost.com/$REPO_NAME/commit/${{ github.event.pull_request.head.sha || github.sha }}/mattermost-enterprise-linux-amd64.tar.gz
+        docker buildx build --push --build-arg MM_PACKAGE=$MM_PACKAGE -t mattermostdevelopment/mm-ee-test:${TAG} --provenance=false .
+  run-performance-bechmarks:
+    uses: ./.github/workflows/performance-benchmarks.yml
+    needs: prepare-docker-build
```

Comment:
Since this is for dot releases, we might want to remove the performance benchmark steps. Those are actually disabled in the monorepo right now since they were incredibly flaky and we haven't had a chance to revisit them

Reply:
Agreed, since we're only doing security and major fixes only, I don't think we need this. It would be fine it it worked correctly but if its flaky, it will hinder the PR merge and waste time. +1 for removing this.

Reply:
👍🏼 Removing